src/foam/nanos/auth/AddressConfig.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.nanos.auth',
+  name: 'AddressConfig',
+
+  ids: ['countryId'],
+
+  properties: [
+    {
+      class: 'Reference',
+      targetDAOKey: 'countryDAO',
+      name: 'countryId',
```

Comment:
rename to country.  our convention for references/relationships. 
as we create  AddressConfig.findCountry(x) which returns the country object.